site/content/docs/developing/storage.md

```diff
@@ -0,0 +1,219 @@
+
+### Manifest Syntax
+The simplest possible EFS volume can be specified with the following syntax:
+
+```yaml
+storage:
+  volumes:
+    myEFSVolume: # This is a variable key and can be set to arbitrary strings.
+      path: '/etc/mount1'
+      efs:
+        id: fs-1234567 
+```
+
+This will create a read-only mounted volume in your service or job's container using the filesystem `fs-1234567`. If mount targets are not created in the subnets of the environment, the task will fail to launch. 
+
+Full syntax for storage follows. 
+
+```yaml
+storage:
+  volumes:
+    {{ volume name }}:
+      path: {{ path at which to mount }} # Required.
+      read_only: {{ boolean }}           # Default: true
+      efs:
+        id: {{ filesystem ID }}          # Required.
+        root_dir: {{ filesystem root }}  # Optional. Defaults to "/". Must not be 
+                                         # specified if using access points.
+        auth: 
+          iam: {{ boolean }}             # Optional. Whether to use IAM authorization when 
+                                         # mounting this filesystem.
+          access_point_id: {{ access point ID}} # Optional. The ID of the EFS Access Point
+                                                # to use when mounting this filesystem.
+```
+
+### Creating Mount Targets
+There are several ways to create mount targets for an existing EFS filesystem: [using the AWS CLI](#with-the-aws-cli) and [using Addons](#with-addons).
+
+#### With the AWS CLI
+To create mount targets for an existing filesystem, you'll need 
+
+1. the ID of that filesystem.
+2. a Copilot environment deployed in the same account and region.
+
+To retrieve the filesystem ID, you can use the AWS CLI:
+```bash
+$ EFS_FILESYSTEMS=$(aws efs describe-file-systems | \
+  jq '.FileSystems[] | {ID: .FileSystemId, CreationTime: .CreationTime, Size: .SizeInBytes.Value}')
+```
+
+If you `echo` this variable you should be able to find which filesystem you need. Assign it to the variable `$EFS_ID` and continue.
+
+You'll also need the public subnets of the Copilot environment and the Environment Security Group. This jq command will filter the output of the describe-stacks call down to simply the desired output value. 
+
+!!!info
+    The filesystem you use MUST be in the same region as your Copilot environment!
+
+```bash
+$ SUBNETS=$(aws cloudformation describe-stacks --stack-name ${YOUR_APP}-${YOUR_ENV} \
+  | jq '.Stacks[] | .Outputs[] | select(.OutputKey == "PublicSubnets") | .OutputValue')
+$ SUBNET1=$(echo $SUBNETS | jq -r 'split(",") | .[0]')
+$ SUBNET2=$(echo $SUBNETS | jq -r 'split(",") | .[1]')
+$ ENV_SG=$(aws cloudformation describe-stacks --stack-name ${YOUR_APP}-${YOUR_ENV} \
+  | jq -r '.Stacks[] | .Outputs[] | select(.OutputKey == "EnvironmentSecurityGroup") | .OutputValue')
+```
+
+Once you have these, creating mount targets is simple. 
```

Comment:
nit: maybe we shouldn't say simple here :D since it's a lot of work to get to this point.
```suggestion
Once you have these, create the mount targets: 
```